src/main/webapp/i18n/en/programmingExercise.json

```
@@ -396,7 +397,7 @@
                 "complaintOnAutomaticAssessment": "Complaint on Automatic Assessment",
                 "complaintOnAutomaticAssessmentTooltip": "Students can complain after receiving an automatic assessment. Which will be evaluated by a tutor afterwards.",
                 "exampleSolutionPublicationDateTooltip": "The date when the Solution Repository becomes available to download for students. Leave blank to disable.",
-                "releaseTestsWithExampleSolution": "Publish tests",
+                "releaseTestsWithExampleSolution": "Publish tests with example solution",
```

Maybe something like this to make clear that students will download both at once.
```suggestion
                "releaseTestsWithExampleSolution": "Include tests into example solution",
```